### PR TITLE
use continuation-local-storage to store session info

### PR DESCRIPTION
**Pull request by Eskibear**
About implementation of `continuation-local-storage`, refer to:
https://github.com/nodejs/node-v0.x-archive/issues/5243

Now simply use:
```
TelemetryWrapper.registerCommand("commandName",
        (args: any[]): void => {
            // TODO
            TelemetryWrapper.currentSession().id;
        }
);
```

### CODE REVIEW

**Comment by Eskibear** [comment]
@akaroml

**Comment by Eskibear** [comment]
@ansyral

File: src/TelemetryWrapper.ts [hunk 1]
```diff
@@ -60,6 +69,10 @@ export module TelemetryWrapper {
         }
     }
 
+    export function currentSession() {
```
Comment (akaroml) [inline src/TelemetryWrapper.ts]: Currently, the session and reporter are in one class. They have different concerns and roles so I would suggest that we separate them.

Comment (Eskibear) [reply src/TelemetryWrapper.ts]: @akaroml If I got it right, they're not defined in one class.
Session is defined in `Session.ts`
TelemetryReporter is defined in package `vscode-extension-telemetry`.

`TelemetryWrapper.ts` defines static variables and functions, e.g. a static TelemetryReporter instance,  and a static method `currentSession()` .

Comment (Eskibear) [reply src/TelemetryWrapper.ts]: got that, `Session` and `TelemetryReporter` decoupled.

**Pull request merged**